lucene/sandbox/src/java/org/apache/lucene/document/FloatPointNearestNeighbor.java

```diff
@@ -75,132 +73,75 @@ public String toString() {
     final int topN;
     final PriorityQueue<NearestHit> hitQueue;
     final float[] origin;
-    private int dims;
-    private int updateMinMaxCounter;
-    private float[] min;
-    private float[] max;
-
+    final private int dims;
+    double worstNearestDistanceSquared = Double.POSITIVE_INFINITY;
+    int worstNearestDistanceDoc =Integer.MAX_VALUE;
```

Comment:
missing space after the equal sign

Reply:
also maybe call it "bottom" (as in bottom of the priority queue) instead of "worst", which suggests it is the worst distance across all docs?

Reply:
renamed